packages/teams-js/src/public/search.ts

```diff
@@ -17,6 +29,12 @@ export namespace search {
   const onClosedHandlerName = 'search.queryClose';
   const onExecutedHandlerName = 'search.queryExecute';
 
+  /** Search Function Messages */
```

Comment:
This doesn't appear to be used anywhere, and exporting something like this makes it callable by external developers and creates a documentation page for it, which probably isn't what you want. Generally, we don't create a const holder like this, but I'm not *necessarily* opposed. I would suggest that you delete this, but if you want to keep it please do the following:

1) Use the values where you are passing messages
2) Add a @hidden comment above this const so no documentation is generated
3) Add *all* of the search messages that go up to the host to this const instead of just one.

Reply:
(Like I said though, if I were you I would just move the const string up to line 31 to be with the rest of the consts)

Reply:
That was for a previous version for test it is unused now will delete it